tests/integration/test_patches.py

```diff
@@ -8,6 +8,21 @@
 from pathlib import Path
 from packit.patches import PatchGenerator, PatchMetadata
 
+TESTS_DIR = str(Path(__file__).parent.parent)
+
+
+def check_copytree_dirs_exists_support():
+    """
+    Old version of shutil.copytree does not support
+    dirs_exist_ok parameter.
+    """
+    if (
+        hasattr(shutil.copytree, "__defaults__")
+        and len(shutil.copytree.__defaults__) >= 5
+    ):
```

Comment:
this can be done pretty nicely with the inspect module: https://docs.python.org/3/library/inspect.html#inspect.getargspec

Reply:
I know, but I don't want to import `inspect` module there. And this seems to be sufficient. Especially as I've met more troubles with inspect module when using various version of Python.